WordPress/src/test/java/org/wordpress/android/ui/mysite/MySiteViewModelTest.kt

```diff
@@ -443,6 +463,42 @@ class MySiteViewModelTest : BaseUnitTest() {
         verify(currentAvatarSource).refresh()
     }
 
+    @Test
+    fun `given mySiteDashboardPhase2FeatureConfig is enabled, when refresh triggers, then all sources are refreshed`() {
+        whenever(mySiteDashboardPhase2FeatureConfig.isEnabled()).thenReturn(true)
+
+        viewModel.refresh()
+
+        allRefreshedMySiteSources.filterIsInstance(MySiteRefreshSource::class.java).forEach { verify(it).refresh() }
+    }
+
+    @Test
+    fun `given mySiteDashboardPhase2FeatureConfig is disabled, when refresh triggers, then select sources refresh`() {
+        whenever(mySiteDashboardPhase2FeatureConfig.isEnabled()).thenReturn(false)
+
+        viewModel.refresh()
+
+        selectRefreshedMySiteSources.filterIsInstance(MySiteRefreshSource::class.java).forEach { verify(it).refresh() }
+    }
+
+    @Test
+    fun `given mySiteDashboardPhase2FeatureConfig is enabled, when refreshing, then refresh indicator should show`() {
```

Comment:
1. Minor (🔍): Consider changing the `given ... , when refreshing, then ...` to `given ... , when are sources refreshing, then ...`
2. Suggestion (💡): Consider rewriting this test with a `given/when/then` structure, like the below:

```
    @Test
    fun `given mySiteDashboardPhase2FeatureConfig is enabled, when are sources refreshing, then refresh indicator should show`() {
        whenever(mySiteDashboardPhase2FeatureConfig.isEnabled()).thenReturn(true)
        allRefreshedMySiteSources.filterIsInstance(MySiteRefreshSource::class.java).forEach {
            whenever(it.isRefreshing()).thenReturn(true)
        }

        val result = viewModel.shouldHideRefreshIndicator()

        assertFalse(result)
    }
```

3. Suggestion (💡): Consider writing another accompanying test for the case when `whenever(it.isRefreshing()).thenReturn(false)` to cover both cases.

Reply:
Just me saving lines, I'll break into two. :(

Reply:
in 340192b

Reply:
🤔  It's late and I'm drawing a blank on the correct way to break this test name onto multiple lines. Tomorrow.

Reply:
@ParaskP7 - ha, you changed the text of these tests in that patch and shortened the name. All good as of 9df2915

Reply:
Thank you @zwarm !

> ha, you changed the text of these tests in that patch and shortened the name.

Yes, you don't need to have the full feature flag name, just a name that you can distinguish is enough. This is what made the difference... 😅